packages/core/helper-plugin/src/utils/fetchClient.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance } from 'axios';
+import qs from 'qs';
+
+import { auth } from './auth';
+
+const fetchClient = (): AxiosInstance => {
+  const instance = axios.create({
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    paramsSerializer: (params) => {
+      return qs.stringify(params, { encode: false });
+    },
+  });
+
+  // Add a request interceptor to add authorization token to headers, rejects errors
+  instance.interceptors.request.use(
+    async (config) => {
+      config.headers.Authorization = `Bearer ${auth.getToken()}`;
+
+      return config;
+    },
+    (error) => Promise.reject(error)
+  );
+
+  // Add a response interceptor, let pass the response, and if error is 401, clear the app storage and reload the page
```

Comment:
Isn't it either / or? 🤔 

```suggestion
  // Add a response interceptor to return the response or handle the error
```